app-services/secrets/README.md

```diff
@@ -16,29 +16,53 @@ Vault needs to be configured with secrets in order to run this example.
 
    ` docker-compose -f docker-compose.yml up`
 
-2. Use docker exec to run commands on the running vault container instance.
+2. Find the root token for Vault in the logs of the container:
```

Comment:
This is required if running the example local, not docker, correct? If so, state something like `If running this example native (not in docker) then you will need to manually set the current Vault token in configuration.`